.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,5 +5,19 @@
 ## Related PR/s
 
 ## Checklist
-
 - [ ] I tested and it works
+
```

Comment:
very nice!

Reply:
yeah, it helped me in this PR actually :)